Makefile

```diff
@@ -111,6 +171,9 @@ BIN := build/bin
 OBJ := build/obj
 INC := build/include/sameboy
 LIBDIR := build/lib
+PKGCONF_DIR := $(LIBDIR)/pkgconfig
+LIBTOOL_LIBRARY := $(LIBDIR)/libsameboy.la
```

Comment:
You absolutely should not create a .la file if you aren't using GNU libtool.

Reply:
I am using libtool.